src/runtime/auto-prompt-manager.ts

```diff
@@ -306,45 +302,24 @@ export class AutoPromptManager {
     )!;
 
     let potentialAction;
-    if (this.actionOrchestrationExperiment_ && !!article.actionOrchestration) {
```

Comment:
@oyj9109 just to confirm, with this cleanup we no longer have any fallback if `article.actionOrchestration` is not present.

Reply:
That should not be a reachable state, but I think it would be still safer to have some bare minimum fallback logic especially for paywalled content in case something unexpected happens.
WDYT about fallback to subscription if (`actionOrchestration` is not present && ActionConfig for subscription is present && content_type==closed)?